tutorials/run-symfony-on-appengine-flexible/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Run Symfony on Google App Engine Flexible Environment
+description: Learn how to deploy a Symfony app to Google App Engine flexible environment.
+author: bshaffer
+tags: App Engine, Symfony, PHP
+date_published: 2017-03-15
+---
+## Symfony
+
+> [Symfony][symfony] is a set of PHP Components, a Web Application framework, a
```

Comment:
_Components_, _Web Application_, _Philosophy_, and _Community_ don't need to be capitalized because they are not on the Symfony site.